salsah/src/public/js/jquery.htmleditor.js

```diff
@@ -138,13 +138,18 @@
 
                 // specify allowed elements, attributes, and classes
                 // this must conform to the `STANDARD_MAPPING`
-                var filter = ' p em strong strike u sub sup; a[!href](salsah-link)';
+                var filter = ' p em strong strike u sub sup hr h1 h2 h3 h4 h5 h6 pre table tbody tr td ol ul li cite blockquote code; a[!href](salsah-link) ';
```

Comment:
I noticed that some of the new elements are not visible in display mode (when content is not being edited). Is there CSS missing? `03_showval.js` handles the display of existing values.